src/TwigYard/Component/LoggerFactory.php

```diff
@@ -50,22 +51,22 @@ class LoggerFactory
 
     /**
      * LoggerFactory constructor.
-     * @param $appRoot
-     * @param $logDir
-     * @param $logOnLevel
-     * @param $logRotationEnabled
-     * @param int $maxFiles
-     * @param null $logglyToken
-     * @param array $logglyTags
+     * @param string $appRoot
+     * @param string $logDir
+     * @param string $logOnLevel
+     * @param bool $logRotationEnabled
+     * @param int|null $maxFiles
+     * @param null|string $logglyToken
```

Comment:
Could You place |null at the end of the typehint for better readability and consistency?

Reply:
Fixed